packages/radio-group/src/LionRadioGroup.js

```diff
@@ -10,4 +10,26 @@ export class LionRadioGroup extends ChoiceGroupMixin(FormGroupMixin(LitElement))
     super.connectedCallback();
     this.setAttribute('role', 'radiogroup');
   }
+
+  /**
+   * @override FormatMixin, during a reset if the current checked value is behind
+   * the initial checked value, they both got unchecked
+   */
```

Comment:
This is an override of the FormGroupMixin I believe. I am wondering why it is necessary.

`child.reset()` should call ---> LionField.reset(), because LionRadio is a LionField extension. Here it does:

```js
reset() {
  this.modelValue = this._initialModelValue;
  this.resetInteractionState();
}
```
So my question would be, why is the initialModelValue "wrong" for radios? And maybe that's where we should implement a fix

Reply:
Yes, I quoted the wrong mixin..

The `_initialModelValue` is correct, the problem is that at some point you get 2 checked items (one restored from initial value and one still checked). Radio group can only have 1 selected child, which causes that the initial value gets deselected again once the selected child gets a reset.

Reply:
Ah yeah fair enough that's a bit tricky